packages/ontario-frontend-cli/resources/templates/ontario-create-app/tests/e2e/search.spec.njk

```diff
@@ -0,0 +1,60 @@
+{# Search End to End and integration tests to check if the search takes us to the expected ontario.ca URL #}/**
+ * E2E tests that checks the functionality of the search bar.
+ */
+
+/**
+ * Helper function to check if element exists and is visible
+ * @param {string} selector - Selector value, either a class or an id, of the element
+ * @param {string} description - Description of the element
+ */
+function checkElement(selector, description) {
+  browser.assert
+    .elementPresent(selector, `${description} exists`)
+    .assert.visible(selector, `${description} is visible`);
+}
```

Comment:
Do these functions have to be redefined per test? Or can they be created in a "Tests Util" folder and imported for use?